src/features/instance/operations/queries/getComponents.ts

```diff
@@ -30,7 +31,7 @@ type GetComponentsResponseWithMetaData = GetComponentsResponse & {
 	key?: string;
 	path?: string;
 	project?: string;
-	readOnly?: boolean;
+	pkg?: string;
```

Comment:
By tracking the package name, instead of coercing it to a bool, we can redeploy regardless of what file is selected within the package. I used "pkg" so we could destructure, and avoid overlap with the "package" name itself.